diff --git a/CMakeLists.txt b/CMakeLists.txt
index b83fbbd7..942cdb87 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -2639,15 +2639,15 @@ if(WIN32 OR CYGWIN OR MSYS)
         # as that's where it expects it to be.
         #
         install(TARGETS ${LIBRARIES_TO_INSTALL}
-                RUNTIME DESTINATION bin/x64
-                LIBRARY DESTINATION lib/x64
-                ARCHIVE DESTINATION lib/x64)
+                RUNTIME DESTINATION bin
+                LIBRARY DESTINATION lib
+                ARCHIVE DESTINATION lib)
         if(NOT MINGW)
             install(FILES $<TARGET_FILE_DIR:${LIBRARY_NAME_STATIC}>/${LIBRARY_NAME_STATIC}.pdb
-                    DESTINATION bin/x64 OPTIONAL)
+                    DESTINATION bin OPTIONAL)
             if(BUILD_SHARED_LIBS)
                 install(FILES $<TARGET_PDB_FILE:${LIBRARY_NAME}>
-                        DESTINATION bin/x64 OPTIONAL)
+                        DESTINATION bin OPTIONAL)
             endif(BUILD_SHARED_LIBS)
         endif(NOT MINGW)
     else(MSVC AND CMAKE_SIZEOF_VOID_P EQUAL 8)
